microsoft-edge/webview2/release-notes.md

```diff
@@ -19,13 +19,32 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > WebView2 bug fixes are either Runtime-specific or SDK-specific.
 
-
 ## Minimum version of the browser or Runtime to load WebView2
 
 To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime is 86.0.616.0.  The minimum version to load WebView2 only changes when a breaking change occurs in the web platform.
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features][SwitchToPreviewChannel].
 
+## 1.0.992.28
+
+Release Date: September 27, 2021
+
+[NuGet package for WebView2 SDK 1.0.992.28][NuGetGallery1.0.992.28]
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 94.0.992.28 or higher.
+
+### General
+
+#### Bug Fixes
+*   Fixed inconsistent crash in WebView2 process when opening a new window.
```

Comment:
This bug fix is not available until build 1026+. I wonder if it'd be misleading in saying this bug fix it's available in 94 WebView2 Runtime?

Reply:
you're right, removed for now.